### PR TITLE
Bug/standardize forms filter incidents

### PR DESCRIPTION
**Pull request by Ethan-Edmond**
# Description

[Explanation Video](https://www.loom.com/share/ecd2d3663b7b42b0a0815b2df81f544a)
Standardizes the filter form on the incidents page to match previous form standardization PRs. I changed the Incidents.css file to a less file and fixed the old broken tests.

## Type of change

Please delete options that are not relevant.

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [X] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

- [X] `npm test`

# Checklist

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes
- [X] There are no merge conflicts

### CODE REVIEW

File: src/components/incidents/Incidents.js [hunk 1]
```diff
@@ -39,23 +31,17 @@ const Incidents = () => {
...
   const [data, setData] = useState([]); // State for User Searches
-  const [selectedTags, setSelectedTags] = useState(['All']);
-  const [queryString, setQueryString] = useState('');
   const [selectedIncidents, setSelectedIncidents] = useLocalStorage(
     'marked',
     []
   ); // all marked incidents saved in local storage
   const [added, setAdded] = useState([]); // data where all checked cases stored(from checkboxes)
-  const [rank, setRank] = useState('All');
+  const [rank, setRank] = useState('');
```
Comment (lgarcia2689) [inline src/components/incidents/Incidents.js:39]: Cleaning this up was a good touch

**Review by lgarcia2689 — APPROVED** [review]
Great Video explaining all the changes that you made, great looking code from the video as well. Looks like you took a lot of time with the design

**Review by austin-carman — APPROVED** [review]
Changes look good, I like that the form is standardized to the other forms being used elsewhere. Form looks very clean, much easier to read than before

File: src/components/incidents/Incidents.js [hunk 2]
```diff
@@ -265,95 +239,22 @@ const Incidents = () => {
...
-          <label htmlFor="dates" className="labels">
-            Date
-            <br></br>
-            <RangePicker
-              onCalendarChange={onDateSelection}
-              className="form-inputs"
-            />
-          </label>
-        </form>
+        <FilterForm onValuesChange={onFilterChange} />
```
Comment (kcjerrell) [inline src/components/incidents/Incidents.js:257]: Thanks for working on this file. It's the longest/messiest file in the whole project.

File: src/components/incidents/FilterForm.js [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+  'Tennessee',
+  'Texas',
+  'Utah',
+  'Vermont',
+  'Virginia',
+  'Washington',
+  'West Virginia',
+  'Wisconsin',
+  'Wyoming',
+];
```
Comment (kcjerrell) [inline src/components/incidents/FilterForm.js]: What about Washington DC?

In src\components\graphs\assets\bargraphAssets.js there's already a list of states. You could export/import that and use ```states.map(s => s.name)``` to get the states.

**Review by kcjerrell — APPROVED** [review]
Thanks for working on the Incidents.js file. It's the longest/messiest file in the whole project.

File: src/components/incidents/FilterForm.js (hunk 3 above)
Comment (Ethan-Edmond) [reply src/components/incidents/FilterForm.js]: Sure I can do that.

**Pull request merged**